src/app/shared/pipes/access-role.pipe.spec.ts

```diff
@@ -0,0 +1,49 @@
+/* @format */
+import { AccessRolePipe } from './access-role.pipe';
+
+describe('GetAltTextPipe', () => {
+  let pipe: AccessRolePipe;
+
+  beforeEach(() => {
+    pipe = new AccessRolePipe();
+  });
+  it('create an instance', () => {
+    expect(pipe).toBeTruthy();
+  });
+
+  it('returns the correct access role for the viewer', () => {
+    const value = 'access.role.viewer';
+    const result = pipe.transform(value);
+    expect(result).toEqual('Viewer');
```

Comment:
Can you fix the lint warnings in this file? You only have to do them in the new files and changes.